src/subcategories/onlineActivity.ts

```diff
@@ -3,7 +3,7 @@ import { makeEnum } from '@transcend-io/type-utils';
 export const OnlineActivitySubCategory = makeEnum({
   // TODO: https://transcend.height.app/T-14003 - add more subcategories
   /** Fallback subcategory */
-  OnlineActivity: 'OnlineActivity',
+  OnlineActivity: 'onlineActivity',
```

Comment:
how come we aren't making these CONSTANT_CASE?

Reply:
im not entirely sure what we want to do here, feel free to make the call but here is the background:


as a general ["style guide"](https://www.notion.so/transcend/4ef10ad243b746d9b2a84f8bb4a1b01a?v=8eb2ce8c21d54b43a916e7f93a563950&p=a6fa0a5f94054c65aafb71221b72243e) thing we normally use CONSTANT_CASE.

im guessing @MadDataScience picked camelCase because thats what IdentifierType was. 

IdentifierType was an exception to the standard because the identifiers are really frequently used in code and displayed on the UI. In the db the identifier.type (IdentifierType) === identifier.name, and identifier.name is shown on the ui, alongside other custom identifier like `username`, `myCustomId` etc... 

for this situation, im not sure we need to break the exception. im not sure whether we are going to end up rendering these values on the UI, or mapping them to a `defineMessages` configuration. id lean towards the latter since it's more intl friendly, and having translations for these defaults would be valuable some day.

Reply:
> im guessing @MadDataScience picked camelCase because thats what IdentifierType was.

That's true. I saw both and it seemed arbitrary, so I picked camelCase, but I don't really have a principled reason for that except that I don't like getting yelled at. 😄  But seriously, if CONSTANT_CASE it's what's in the style guide, then I think we should stick with that unless @LuGary has a better reason in mind.

Reply:
since larry is out il'l take over this PR and update it to CONSTANT_CASE